apps/dashboard/app/views/launchers/editable_form_fields/_editable_text_field.erb

```diff
@@ -7,7 +7,16 @@
 
   <div class="d-none edit-group mb-3">
     <%= render(partial: 'launchers/editable_form_fields/edit_fixed_field',  locals: { form: form, attrib: attrib, fixed: fixed }) %>
-  </div>
+  
+    <% if field_id.match?('auto_environment_variable') %>
+      <label for="aev_name">Name</label>
+      <input class="form-control edit-field" type="text" name="aev_name" id="aev_name"></inut>
+
+      <label for="aev_value">Value</label>
+      <input class="form-control edit-field" disabled=true type="text" name="aev_value" id="aev_value"></input>
```

Comment:
I'm guessing the way this controller is updated is a byproduct of the `launcher#edit` process where the user is editing both the key and the value.

I think we still need `text_field` because that's what'll be rendered when on `launcher#show` pages.  But I don't think we need `value` here.  I think we can get by with just the name input and some javascript to fix the id/name of the value input before it's submitted.

Reply:
I did not see this comment until this morning, but I'm pretty sure I see what you're saying. I'm seeing:

When a user adds a new environment variable, they have a disabled field labeled 'Environment Variable' - they have to click 'edit' to get the 'name' field, then once they enter something in the name field, they can edit the value regardless of whether the edit portion of the form field box is open or not. But they have to click 'edit' to edit the variable name.

Same page?

Reply:
> Same page?

Yes I think so. You had 3 fields when I think we only need 2.

> But they have to click 'edit' to edit the variable name.

We can move forward with what you have, but I do wonder about the UX here, so there may be changes in the future. What I'm wondering is should the user click the edit button to see the variable name or the variable value? Or should it be special in that the user kinda needs to edit both?